### PR TITLE
chore(dart): remove client packages

### PR DESCRIPTION
**Pull request by aallam**
## 🧭 What and Why

- Dart is a language used mainly (almost exclusively) in front end applications context.
- A few APIs are meant to be used from a secure environment.

### Changes included:

Remove `abtesting`, `analytics`, `monitoring`, `personalization`,`ingestion` and `query_suggestion` until a need arises.

### CODE REVIEW

**Comment by millotp** [comment]
maybe you can leave `ingestion` ? it can be used on front or back, like the dashboard

File: config/openapitools.json [hunk 1]
```diff
@@ -181,42 +181,6 @@
...
           "packageVersion": "0.1.2"
         }
-      },
-      "dart-personalization": {
-        "output": "#{cwd}/clients/algoliasearch-client-dart/packages/client_personalization",
-        "additionalProperties": {
-          "packageVersion": "0.1.2"
-        }
-      },
-      "dart-analytics": {
```
Comment (shortcuts) [inline config/openapitools.json:191]: analytics also is front-end related stuff, no?

Comment (aallam) [reply config/openapitools.json:191]: Although it's possible to use these APIs in the frontend, they are not typically utilized to build search experiences in frontend apps. However, we can always add them back if a need arises for some special case we are unaware of.

**Review by shortcuts — APPROVED** [review]
gg lets gooo

**Pull request merged**